<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1049</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FortiCNAPP Extensible Reporting</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_9">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBoxUseCache">
          <property name="text">
           <string>Use Cache</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="labelSelectReport">
          <property name="text">
           <string>Select Report:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxReportSelector"/>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="labelCurrentLogo">
          <property name="text">
           <string>Custom Logo:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditCustomLogo">
          <property name="text">
           <string>None</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonSelectCustomLogo">
          <property name="text">
           <string>Select...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="labelCustomer">
          <property name="text">
           <string>Customer:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditCustomer"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="labelAuthor">
          <property name="text">
           <string>Author:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditAuthor"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="labelVulnStartTime">
          <property name="text">
           <string>Vuln Start Time</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBoxVulnStartTimeDays"/>
        </item>
        <item>
         <widget class="QLabel" name="labelVulnStartTimeDays">
          <property name="text">
           <string>Days and</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBoxVulnStartTimeHours"/>
        </item>
        <item>
         <widget class="QLabel" name="labelVulnStartTimeHours">
          <property name="text">
           <string>Hours Ago</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelVulnEndTime">
          <property name="text">
           <string>Vuln End Time</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBoxVulnEndTimeHours"/>
        </item>
        <item>
         <widget class="QLabel" name="labelVulnEndTimeDays">
          <property name="text">
           <string>Days and</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBoxVulnEndTimeDays"/>
        </item>
        <item>
         <widget class="QLabel" name="labelVulnEndTimeHours">
          <property name="text">
           <string>Hours Ago</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="labelAlertStartTime">
          <property name="text">
           <string>Alert Start Time</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBoxAlertStartTimeDays"/>
        </item>
        <item>
         <widget class="QLabel" name="labelAlertStartTimeDays">
          <property name="text">
           <string>Days and</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBoxAlertStartTimeHours"/>
        </item>
        <item>
         <widget class="QLabel" name="labelAlertStartTimeHours">
          <property name="text">
           <string>Hours Ago</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelAlertEndTime">
          <property name="text">
           <string>Alert End Time</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBoxAlertEndTimeDays"/>
        </item>
        <item>
         <widget class="QLabel" name="labelAlertEndTimeDays">
          <property name="text">
           <string>Days and</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBoxAlertEndTimeHours"/>
        </item>
        <item>
         <widget class="QLabel" name="labelAlertEndTimeHours">
          <property name="text">
           <string>Hours Ago</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="labelReccommendations">
          <property name="text">
           <string>Recommendations Text:</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="plainTextEditRecommendations"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QPushButton" name="pushButtonRunReport">
          <property name="text">
           <string>Run Report</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonWriteHTML">
          <property name="text">
           <string>Write HTML</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonWritePDF">
          <property name="text">
           <string>Write PDF</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonTogglePreview">
          <property name="text">
           <string>Toggle Preview Window</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1049</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
